diff --git a/setup.py b/setup.py
index 3fea215..b7bc353 100644
--- a/setup.py
+++ b/setup.py
@@ -1,5 +1,4 @@
 from setuptools import setup, find_packages
-from pixeloe import cli
 
 
 setup(
@@ -15,8 +14,8 @@ setup(
     install_requires=["opencv-python", "numpy", "pillow"],
     entry_points={
         "console_scripts": [
-            f"pixeloe.{command}=pixeloe.cli:{func}"
-            for command, func in cli.command_map.items()
+            "pixeloe.pixelize=pixeloe.cli:pixelize",
+            "pixeloe.outline=pixeloe.cli:outline"
         ]
     },
     python_requires=">=3.10",
